example/src/main/java/org/xmtp/android/example/conversation/ConversationDetailViewModel.kt

```diff
@@ -56,6 +67,24 @@ class ConversationDetailViewModel(private val savedStateHandle: SavedStateHandle
         }
     }
 
+    @OptIn(ExperimentalCoroutinesApi::class)
+    val streamMessages: StateFlow<List<MessageListItem>> =
+        stateFlow(viewModelScope, emptyList()) { subscriptionCount ->
+            if (conversation == null) {
+                conversation = ClientManager.client.fetchConversation(conversationTopic)
+            }
+            conversation!!.streamMessages()
```

Comment:
I think we can avoid the force unwrap here similar to how we do above. JUST TO BE SAFE lol:

`conversation?.let {
  it.streamMessages()
`